src/components/content/order/common/FlavorSelection.tsx

```diff
@@ -43,19 +83,32 @@ export const FlavorSelection = ({
                                 value={selectFlavor}
                                 className={flavorStyles.antRadioGroup}
                             >
-                                {flavorList.map((flavor: ServiceFlavor) => (
-                                    <div key={flavor.name} className={flavorStyles.customRadioButton}>
-                                        <Radio.Button
-                                            key={flavor.name}
-                                            value={flavor.name}
-                                            className={flavorStyles.customRadioButtonContent}
-                                        >
-                                            {FlavorTitle(flavor.name)}
-                                            <div className={flavorStyles.flavorPriceContent}>{FlavorPrice()}</div>
-                                            {FlavorFeatures(flavor)}
-                                        </Radio.Button>
+                                {getServicePriceQuery.isSuccess ? (
```

Comment:
we must separate the flavor button and the getServicePriceQuery

The flavor box must be displayed immediately and only the price field must be loading until the prices are loaded. 

Also currently no error is shown when price loading fails. We must show flavors without prices. 

In case of failure.. we must just just show an small error tag and on mouse hover we can show the error. We need not put a full big alert here. And also we need a retry option with the tag.